src/AutoRest.CSharp/Common/Output/Models/Method.cs

```diff
@@ -10,8 +10,8 @@ namespace AutoRest.CSharp.Common.Output.Models
     internal class Method
     {
         public MethodSignatureBase Signature { get; }
-        public MethodBodyStatement? Body { get; }
-        public ValueExpression? BodyExpression { get; }
+        public MethodBodyStatement? Body { get; protected init; }
```

Comment:
what is this change for? don't see any derived types of `Method`

Reply:
In a previous commit we have. But now I have reverted that. I will change this back